File: web-deteccion/src/app/royal/modulos/framework-comun/contabilidad/dominio/filtro/FiltroComunAcCostcentergroup.ts
import { DominioPaginacion } from './../../../../../framework/modelo/generico/DominioPaginacion';
 

export class FiltroComunAcCostcentergroup {
    constructor(){
        this.paginacion = new DominioPaginacion();
    }
    
    paginacion: DominioPaginacion;
    
    costcentergroup : string;
    subgrupo:string
    localname : string;
    englishname : string;
    costcentermajorgroup : string;
    status : string;
    lastuser : string;
    lastdate : Date;
    tipoexportar:string
}

File: blinkeye-framewk-comun-dominio/src/main/java/net/royal/spring/contabilidad/dominio/filtro/FiltroComunAcCostcentergroup.java
package net.royal.spring.contabilidad.dominio.filtro;

import net.royal.spring.framework.modelo.generico.DominioPaginacion;
import net.royal.spring.framework.modelo.generico.DominioTransaccion;

public class FiltroComunAcCostcentergroup extends DominioTransaccion implements java.io.Serializable{
	private String costcentergroup;
	private String tipoexportar;
	

	public String getTipoexportar() {
		return tipoexportar;
	}

	public void setTipoexportar(String tipoexportar) {
		this.tipoexportar = tipoexportar;
	}

	/**
	 * 
	 * 
	 * @campo CostCenterGroup
	*/
	public String getCostcentergroup() {
		return costcentergroup;
	}

	/**
	 * 
	 * 
	 * @campo CostCenterGroup
	*/
	public void setCostcentergroup(String costcentergroup) {
		this.costcentergroup = costcentergroup;
	}

	private DominioPaginacion paginacion;

	public DominioPaginacion getPaginacion() {
		if (paginacion == null)
			paginacion = new DominioPaginacion();
		return paginacion;
	}

	public void setPaginacion(DominioPaginacion paginacion) {
		this.paginacion = paginacion;
	}
}
